test/test_helper.exs

```diff
@@ -1,2 +1,22 @@
 ExUnit.configure(exclude: [pending: true], timeout: 30_000)
 ExUnit.start()
+
+fixtures_schemas = [
+  Absinthe.Fixtures.ArgumentsSchema,
+  Absinthe.Fixtures.ColorSchema,
+  Absinthe.Fixtures.ContactSchema,
+  Absinthe.Fixtures.CustomTypesSchema,
+  Absinthe.Fixtures.IdTestSchema,
+  Absinthe.Fixtures.NullListsSchema,
+  Absinthe.Fixtures.ObjectTimesSchema,
+  Absinthe.Fixtures.OnlyQuerySchema,
+  Absinthe.Fixtures.PetsSchema,
+  Absinthe.Fixtures.StrictSchema,
+  Absinthe.Fixtures.TimesSchema,
+  Absinthe.Fixtures.Things.SDLSchema,
+  Absinthe.Fixtures.Things.MacroSchema
+]
+
+for schema <- fixtures_schemas do
+  Absinthe.Schema.Manager.start_link(schema)
```

Comment:
This has to be added otherwise the modules aren't actually available reliably. They _are_ available the first time they're compiled because compiling a module with the persistent term provider DOES put the schema into persistent term, this is how all of the inline schemas in the test files still work.

However for these `.ex` fixtures, after the test suite has been compiled once, re-running tests would fail because those schemas aren't being recompiled, and thus aren't going back in persistent term. Thus, we need to use the genserver.

This won't really be an issue for end users because they should always place the genserver in their supervision tree.